README.md

```diff
@@ -44,30 +44,19 @@ Under the [Users](https://github.com/dynawo/dynawo-docker/tree/master/Users) fol
 $> docker run -it dynawo/dynawo
 ```
 
-Once inside the container the user can use the `dynawo` alias and launch:
+The previous command is a very basic one to just test Dyna&omega;o very quickly. **We recommend** you to create a container by sharing a folder of your host machine so that you can access it inside and outside the container to create your own test cases. You can launch the following command (`FOLDER_PATH` needs to be modified with appropriate value):
 ``` bash
-dynawo_user@contaiderID:dynawo$> dynawo nrt
+$> docker run -it -e LOCAL_USER_ID=`id -u $USER` -e LOCAL_GROUP_ID=`id -g $USER` -v FOLDER_PATH:/home/dynawo_user/SharedFolder dynawo/dynawo
```

Comment:
FOLDER_PATH should exist beforehands or is it created otherwise? If not, it should be written to help user

Reply:
Not sure about that but I think it should exists... Maybe in Windows it is handled well but in Linux I don't know and I never tried... I will add the comment